src/components/contextual/pages/vebal/cross-chain-boost/SyncNetworkAction.vue

```diff
@@ -75,7 +77,9 @@ async function handleAction(network: Network) {
       'tempSyncingNetworks',
       JSON.stringify(tempSyncingNetworks.value)
     );
-
+    setTimeout(() => {
+      setSyncTxHashes(network, tx.hash);
+    }, 5000);
```

Comment:
I think this should be tried more than once. Can call on an interval after the tx success? Maybe allow it to be called 3-5 times every 10s?

Reply:
done